examples/redux-observable-shopping-cart/js/services/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Mocking client-server processing
+ */
+import _products from './products'
+
+const TIMEOUT = 100;
+const MAX_CHECKOUT = 2; // max different items
+
+export const api = {
+  getProducts() {
+    return new Promise( resolve => {
+      setTimeout(() => resolve(_products), TIMEOUT)
+    })
+  },
+
```

Comment:
I'm thinking we should use Observables instead of Promises everywhere to set a good example for consistency/easy/cancelability.

e.g.

```js
getProducts() {
  return new Observable(observer => {
    const timerId = setTimeout(() => {
      observer.next(_products);
      observer.complete();
    }, TIMEOUT);

    return () => clearTimeout(timerId);
  });
}
```

What do you think?